src/org/camunda/latera/bss/connectors/hid/hydra/Account.groovy

```diff
@@ -506,17 +422,17 @@ trait Account {
     }
   }
 
-  Boolean processAccount(
-    def accountId,
-    Temporal beginDate = local(),
-    Temporal endDate   = null
-  ) {
+  Boolean processAccount(Map input = [:], def accountId) {
+    LinkedHashMap params = [
+      beginDate : local(),
+      endDate   : null
+    ] + input
```

Comment:
In groovy 2.4 this is the only way - all named args are passed into first Map argument and they are separated from positional arguments. So you cannot just call `func(def name)` like `func(name: 'abc')`. The only exclusion here is constructors with named args.

In groovy 2.5 there is a better way to do that, there is a `@NamedArg` annotation which can be set for this map fields directly in args description instead of body. But I've got a failure while trying to build .jar with groovy 2.5.